nevergrad/functions/multiobjective/hypervolume.py

```diff
@@ -0,0 +1,303 @@
+import typing as tp
```

Comment:
Can you add the same header as on all other files? This is required to mention that it is released under the MIT license